hack/generator/pkg/config/configuration.go

```diff
@@ -243,12 +244,19 @@ func buildExportFilterFunc(f *ExportFilter, allTypes astmodel.Types) ExportFilte
 
 	case ExportFilterIncludeTransitive:
 		applicableTypes := make(astmodel.TypeNameSet)
+		foundMatch := false
 		for tn := range allTypes {
 			if f.AppliesToType(tn) {
+				foundMatch = true
 				collectAllReferencedTypes(allTypes, tn, applicableTypes)
 			}
 		}
 
+		// TODO: should we consider somehow turning this into an err?
```

Comment:
As long as an error here results in the generator aborting due to the useless filter, then _**yes**_ this should be an error.

Reply:
Fixed.